spec/models/latest_blood_pressures_per_patient_per_month_spec.rb

```diff
@@ -5,10 +5,63 @@ def refresh_views
     described_class.refresh
   end
 
+  def refresh_views_with_pg_set_to_reporting_time_zone
+    original = ActiveRecord::Base.connection.execute("SELECT current_setting('TIMEZONE')").first["current_setting"]
+    ActiveRecord::Base.connection.execute("SET LOCAL TIME ZONE '#{Period::ANALYTICS_TIME_ZONE}'")
+    described_class.refresh
+    ActiveRecord::Base.connection.execute("SET LOCAL TIME ZONE #{original}")
+  end
+
   describe "Associations" do
     it { should belong_to(:patient) }
   end
 
+  describe "handling time zones" do
+    it "stores mat view timestamps in UTC to stay consistent with the source application tables" do
```

Comment:
This spec is long and verbose, but I wanted to try and verify a bunch of scenarios _and_ emulate the different time zone configurations that happen in the app between record creation (UTC via the API) and reporting (IST via our before_actions).